extras/rebalance/directory-rebalance.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python3
+import argparse
+import os
+import errno
+import time
+import sys
+import datetime
+import hashlib
+import logging
+
+def size_fmt(num, suffix='B'):
+    for unit in ['','Ki','Mi','Gi','Ti','Pi','Ei','Zi']:
+        if abs(num) < 1024.0:
+            return "{:3.1f} {}{}".format(num, unit, suffix)
+        num /= 1024.0
+    return "{:.2f} {}{}".format(num, 'Yi', suffix)
```

Comment:
`'Yi'` is fixed ?

nit: given that we are using python3, wouldn't `f"<fmt string>"` be cleaner and shorter ?

```suggestion
    return f"{num:6.2f} {unit}{suffix}"
```

Reply:
Let me try that.